common/src/helpers/router.tsx

```diff
@@ -114,8 +114,9 @@ export const useRouter = (
  *
  * @param imgFile The name of the img file, not including the /images/ path.
  */
-export function imageUrl(baseUrl: string, imgFile: string): string {
-  return `${baseUrl}/public/images/${imgFile}`;
+export function imageUrl(imgFile: string): string {
```

Comment:
I don't think we can can remove the baseUrl parameter entirely. This function is in /common, which means it is used by multiple projects. In the admin project, images should come from `/admin/public/...` while in the talentsearch they come from somewhere different.